sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/implementation/PartitionPumpManager.java

```diff
@@ -126,8 +123,8 @@ public void startPartitionPump(PartitionOwnership claimedOwnership) {
         eventHubConsumer.receive().subscribe(eventData -> {
             try {
                 Context processSpanContext = startProcessTracingSpan(eventData);
-                if (processSpanContext.getData(SPAN_CONTEXT).isPresent()) {
-                    eventData.addContext(SPAN_CONTEXT, processSpanContext);
+                if (processSpanContext.getData("span-context").isPresent()) {
```

Comment:
Use a shared constants class in event hubs so we don't have to redefine these shared strings. There is a ClientConstants class.